core/src/validator.rs

```diff
@@ -119,6 +119,51 @@ use {
 const MAX_COMPLETED_DATA_SETS_IN_CHANNEL: usize = 100_000;
 const WAIT_FOR_SUPERMAJORITY_THRESHOLD_PERCENT: u64 = 80;
 
+pub const DEFAULT_REPLAYING_BACKEND: &str = "blockstore-processor";
+pub const DEFAULT_BANKING_BACKEND: &str = "multi-iterator";
+
+#[derive(Clone, Debug)]
+pub enum ReplayingBackend {
+    BlockstoreProcessor,
+    UnifiedScheduler,
+}
+
+impl Default for ReplayingBackend {
+    fn default() -> Self {
+        Self::from(DEFAULT_REPLAYING_BACKEND)
+    }
+}
+
+impl From<&str> for ReplayingBackend {
+    fn from(string: &str) -> Self {
+        match string {
+            "blockstore-processor" => Self::BlockstoreProcessor,
+            "unified-scheduler" => Self::UnifiedScheduler,
+            bad_backend => panic!("Invalid replaying backend: {bad_backend}"),
+        }
+    }
+}
+
+#[derive(Clone, Debug)]
+pub enum BankingBackend {
+    MultiIterator,
+}
+
+impl Default for BankingBackend {
+    fn default() -> Self {
+        Self::from(DEFAULT_BANKING_BACKEND)
+    }
+}
+
+impl From<&str> for BankingBackend {
+    fn from(string: &str) -> Self {
+        match string {
+            "multi-iterator" => Self::MultiIterator,
```

Comment:
i think this is decent name for current impl. :)

Reply:
hmm, i actually kinda like `thread-local-multi-iterator`. I think longer name is more or less tolerable in this context (very internal flag), if longer name contributes to less ambiguous thing to interested party (devs, veteran node operators).

Reply:
That's fine with me!

Reply:
done: 92d745d7e2630ab963207b666e8e3522e554c143